pkg/consts/consts.go

```diff
@@ -41,7 +43,9 @@ const (
 // Finalizers.
 // =================================================
 
+// System reserved finalizers.
 const (
+	// FinalizerScaleView Deprecated.
```

Comment:
I think we should add the effect that this finalizer has here in the docstring. Do we simply wait unitl `ScaleView` is deleted if we delete a RW instance?